tests/test_integrations.py

```diff
@@ -123,3 +127,20 @@ def test_nested_app_references(self):
         self.assertContains(response, "pagecontent")
         self.assertContains(response, get_object_preview_url(page_content))
         self.assertContains(response, page_content.versions.first().state)
+
+
+class VersioningIntegrationTestCase(CMSTestCase):
+    def test_unpublish_versioned_item(self):
```

Comment:
I'm confused by what this test is actually checking for. The filter covers that when versions are set to certain states they are hidden / shown. What is the purpose / aim of this test?

I would have thought that a test for the state selected would be calling get_all_reference_objects

Reply:
@Aiky30 Removed the test as we already have a test that confirms unpublishing still works as expected 

https://github.com/Bernardvdv/djangocms-references/blob/bug/unpublish_parameter_updated/tests/test_cms_config.py#L142